### ISSUE TITLE
Bug/issue list of Alert/Logging

### ISSUE THREAD

**Issue by vimniky** [rancher/rancher#11562]
## Bug 1

Searchable-select won't work right when the `content prop` dynamic changes as a whole, which would cause bugs in the project new/edit alert page in the following situation:

1. Go to: project -> tools -> alerts -> new/edit
2. Check the  `workload`  radio.  
3. Choose a targetType (one of Deployment/Daemonset/Statefulset)
4. The `source type` Dropdown should show the right resource list. 
5. But because this bug, the `source type` Dropdown won't show the right resource list once you changed the `targetType`.

Aaron already handled this bug in his pipeline PR and this bug should be fixed once the pipeline PR is merged.

## Bug 2

Can't remove exist nodeSelector/workloadSelector/logging outputTag. I think this is a known bug.

## Bug 3

For now, doing alert action(mute/unmute/activate/deactivate) will throw an error. This bug will be fixed once @zionwu 's update got merged.

## Bug4

The `Condition`(pod is restarted x times in y minutes) field of the project alert table display is wrong. This will be fixed as new PR is merged.

**Comment by zionwu** [comment]
Bug 3 is fixed in PR https://github.com/rancher/rancher/pull/11542

**Comment by tfiduccia** [comment]
Version - 2.0 master 2/23
Verified fixed
